### PR TITLE
[Enhancement] fix sonar problems

### PR DESCRIPTION
**Pull request by packy92**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Fix sonar problems
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 10 (30.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|0|2|00.00%|[291, 331]|
|:large_blue_circle:|com/starrocks/common/util/BrokerUtil.java|0|2|00.00%|[572, 573]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[183]|
|:large_blue_circle:|com/starrocks/http/HttpServerHandler.java|0|1|00.00%|[119]|
|:large_blue_circle:|com/starrocks/catalog/Function.java|2|3|66.67%|[622]|
|:large_blue_circle:|com/starrocks/transaction/DatabaseTransactionMgr.java|1|1|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 10 (30.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|0|2|00.00%|[291, 331]|
|:large_blue_circle:|com/starrocks/common/util/BrokerUtil.java|0|2|00.00%|[572, 573]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[183]|
|:large_blue_circle:|com/starrocks/http/HttpServerHandler.java|0|1|00.00%|[119]|
|:large_blue_circle:|com/starrocks/catalog/Function.java|2|3|66.67%|[622]|
|:large_blue_circle:|com/starrocks/transaction/DatabaseTransactionMgr.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 1]
```diff
@@ -71,7 +71,7 @@
 import com.starrocks.server.GlobalStateMgr;
 import com.starrocks.thrift.TUniqueId;
 import io.opentelemetry.api.trace.Span;
-import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections4.CollectionUtils;
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:74]: Any difference? Do we need to update all import?

**Review by murphyatwork — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java (hunk 1 above)
Comment (packy92) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:74]: Yes, better to use collections4

**Comment by packy92** [comment]
run starrocks_be_unittest

**Pull request merged**